crates/astria-conductor/src/conductor.rs

```diff
@@ -118,15 +121,21 @@ impl Conductor {
             tasks.spawn(Self::SEQUENCER, sequencer_reader.run_until_stopped());
             shutdown_channels.insert(Self::SEQUENCER, shutdown_tx);
             sync_done_rx
+        } else {
+            let (sync_done_tx, sync_done_rx) = oneshot::channel();
```

Comment:
This is going to lead to some very confusing logs: in the loop `sync_done_rx.recv()` will trigger immediately, notifying the user that a thing that was never supposed to run supposedly ran anyway.

Please change this to this instead:
```rust
let mut sync_done = futures::future::Fuse::terminated();
if cfg.execution_commit_level != FirmOnly {
    todo!("set up stuff");
    sync_done = sync_done_rx.fuse();
}
```
and then redefine `sync_done: Fuse<oneshot::Receiver<()>>` in the conductor definition (this is currently done as the first in `run_until_stopped`, but no longer necessary).